src/Config.cpp

```diff
@@ -20,9 +20,9 @@
 
 #include "AtomicFile.hpp"
 #include "Compression.hpp"
+#include "SloppinessEnum.hpp"
```

Comment:
Having "Enum" in the filename feels a bit redundant. Would you be OK with naming it `Sloppiness.hpp`?

Reply:
Yeah that's strange, but there is a huge number of files in src and it's difficult to find what you are looking for. Originally I did this because of `Statistic` and `Statistics`.
Should I rename them?

Reply:
renamed. see latest version.

Reply:
> huge number of files in src

We must be coming from quite different realities, because to me ccache is a small project and has relatively few source code files. Compare with larger projects like (just spontaneously picking a few) CMake, Git, Linux and glibc. I'm not saying that it's bad to group source code files, I just don't think the number of files in the src directory is spectacular in any way.